src/css/page_sections/_LanguageSelectBar.scss

```diff
@@ -18,6 +18,12 @@
   }
 }
 
+.coa-rtl {
+  .coa-LanguageSelectBar__language-list {
+    flex-direction: row-reverse;
```

Comment:
I was going to wait and see how this worked out in usability testing first before changing. I asked @cthibaultatx on Friday about this and she mentioned that there might be imbalance visually if the language select stayed on the same side. Either way, maybe its worth asking again now that its implemented this way? 🤷‍♂️